Sources/Interpreter/Interpreter.swift

```diff
@@ -152,7 +152,11 @@ public struct Interpreter {
       jump(to: x.target)
       return
     case let x as Call:
-      _ = x
+      let p = x.arguments.map { address($0)! } + [address(x.output)!]
+      let r = try nextCodePointer()
+      callStack.append(.init(returnAddress: r, parameters: p))
+      jump(to: (x.callee.constant as! FunctionReference).function)
+      return
```

Comment:
Experimenting with naming; I'm not committed to any of these changes, except for that I really want to see `callStack.push`.

```suggestion
      let calleeIO = x.arguments.lazy.map { address($0)! } + [address(x.output)!]
      try callStack.push(
        StackFrame(returnAddress: nextCodePointer(), parametersAndResult: calleeIO))
      jump(to: (x.callee.constant as! FunctionReference).function)
      return
```

Why does `nextCodePointer()` throw?  Seems like any UB would have been caught by the time the current program counter was set.